contracts/Tokens/VTokens/VBep20.sol

```diff
@@ -49,7 +49,25 @@ contract VBep20 is VToken, VBep20Interface {
     // @custom:event Emits Transfer event on success
     // @custom:event Emits RedeemFee when fee is charged by the treasury
     function redeem(uint redeemTokens) external returns (uint) {
-        return redeemInternal(redeemTokens);
+        return redeemInternal(msg.sender, msg.sender, redeemTokens);
+    }
+
+    /**
+     * @notice Sender redeems assets on behalf of some other address. This function is only available
+     *   for senders, explicitly marked as delegates of the supplier using `comptroller.updateDelegate`
+     * @dev Accrues interest whether or not the operation succeeds, unless reverted
+     * @param redeemer The user on behalf of whom to redeem
+     * @param redeemTokens The number of vTokens to redeem into underlying
+     * @return uint Returns 0 on success, otherwise returns a failure code (see ErrorReporter.sol for details).
+     */
+    // @custom:event Emits Redeem event on success
+    // @custom:event Emits Transfer event on success
+    // @custom:event Emits RedeemFee when fee is charged by the treasury
```

Comment:
Shouldn't we be consistent with the format of the natspect comments? (maybe we cannot be, not sure)

Reply:
We dis this because, unlike in other repositories that support the 0.8 Solidity version, the **@custom:event** netspec comment format is not compatible with the 0.5 Solidity version. Therefore, we made this change to ensure proper compilation, as attempting to use the format from 0.8 Solidity version files would result in a compilation error.